android/lib/build.gradle

```diff
@@ -501,32 +505,76 @@ publishing {
 }
 
 project.afterEvaluate {
+    // Include Java variant definition in Gradle Module Metadata
+    def javaApiConfig = configurations.create("javaApi").tap {
+        canBeResolved = false
+        canBeConsumed = true
+        attributes {
+            attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category.class, Category.LIBRARY))
+            attribute(TargetJvmEnvironment.TARGET_JVM_ENVIRONMENT_ATTRIBUTE, objects.named(TargetJvmEnvironment.class, TargetJvmEnvironment.STANDARD_JVM))
+            attribute(KotlinPlatformType.attribute, KotlinPlatformType.jvm)
+            attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage.class, Usage.JAVA_API))
+        }
+    }
+
+    // Add variant artifact as a dependencies, since it's in another module
+    dependencies {
```

Comment:
Is this going to work for both the CE and EE versions?

Reply:
All of this should work the same for both CE and EE, just with the variant artifacts being different for EE. So
```
javaApi("com.couchbase.lite:couchbase-lite-java-ee:$version")
```
here and
```
androidApi("com.couchbase.lite:couchbase-lite-android-ee:$version")
```
in the Java lib's **build.gradle**.

It might make sense to use a variable, like `CBL_ANDROID_LIB`, as is used in other places currently.